sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/HiveScriptTransformationExec.scala

```diff
@@ -0,0 +1,400 @@
+/*
```

Comment:
@AngersZhuuuu, let's rename this file later when you actually add `ScriptTransformationExec`. If we do the renaming first in this PR and merge, I think that will keep file renaming.